nautobot/core/views/generic.py

```diff
@@ -205,13 +205,12 @@ def get(self, request):
         filter_form = None
         hide_hierarchy_ui = False
         clear_view = request.GET.get("clear_view", False)
+        resolved_path = resolve(request.path)
+        view_name = f"{resolved_path.app_name}:{resolved_path.url_name}"
```

Comment:
I'd suggest changing `view_name` to `list_url` for clarity if that doesn't cause any problems.